src/plugins/controls/public/control_types/options_list/options_list_strings.ts

```diff
@@ -15,7 +15,7 @@ export const OptionsListStrings = {
     }),
   getDescription: () =>
     i18n.translate('controls.optionsList.description', {
-      defaultMessage: 'Add control that allows options to be selected from a dropdown.',
+      defaultMessage: 'Allows options to be selected from a dropdown.',
```

Comment:
@KOTungseth Your help would be appreciated here - looking for a way to describe exactly what an options list control is for the tooltip 🙇 I don't have the range slider in this PR yet, but you could also tackle that here for @cqliu1's reference?

Reply:
How about:

`Add a dropdown of selectable fields`

or

`Add a dropdown of fields for applying dashboard-level filters`

cc: @gchaps

Reply:
One thing about this verbiage, the `fields` are not selectable, the field's `value`s are selectable.

so maybe:

`Add a dropdown of selectable values from a field`?

Reply:
I like `Add a dropdown of selectable values from a field`! I think if we start getting in to `dashboard-level filters`, it might get confusing - but perhaps we could somehow add a tooltip or info icon that describes what controls do *in general*, which is where we could talk about that? That should be part of a separate PR though

Reply:
How about using "menu" instead of "dropdown::

> Add a menu for selecting field values

Reply:
Oh! `Add a menu for selecting field values` sounds great 😄